lib/repl.js

```diff
@@ -772,6 +772,10 @@ function REPLServer(prompt,
         const ln = lines.length === 1 ? ' ' : ':\n';
         errStack = `Uncaught${ln}${errStack}`;
       }
+      // If colors are allowed, print in red
+      if (self.useColors) {
+        errStack = '\u001b[31m' + errStack + '\u001b[39m';
```

Comment:
what if the stack already includes colors? 
also please use `internal/util/colors`

Reply:
If the stack already includes colors, this breaks. I don't think I'll end up merging this PR, as I'm trying to merge <https://github.com/nodejs/repl/> into the main NodeJS, which will redo all of this anyway.

See #52510 and <https://github.com/nodejs/repl/pull/54>